JsonClient/JsonClient/JsonClient.cs

```diff
@@ -53,38 +53,42 @@ public JsonClient()
             }
         }
 
-    //    public async Task<Comment> postComment(Client client, Comment comment){
-    //         //post Comment
-    //         try{
-    //             HttpResponseMessage response = await client.PostAsJsonAsync("comments", comment);
-    //             response.EnsureSuccessStatusCode();
+        public async Task<TModel> PostAsync(TModel model){
+             //post Comment
+             try{
+                 var requestURL = $"{typeof(TModel).Name.ToLower()}s";
+                 HttpResponseMessage response = await client.PostAsJsonAsync(requestURL, model);
+                 response.EnsureSuccessStatusCode();
 
-    //             Comment comments = await response.Content.ReadAsAsync<Comment>();
-    //             Console.WriteLine("saved");
-    //             return comments;
+                 var models = await response.Content.ReadAsAsync<TModel>();
+                 Console.WriteLine("saved");
+                 return models;
 
-    //         }
-    //         catch(HttpRequestException e){
-    //             Console.WriteLine("{0}", e.Message);
-    //             return null;
-    //         }
-    //     }
+             }
+             catch(HttpRequestException e){
+                 Console.WriteLine("{0}", e.Message);
+                 return default(TModel);
+             }
+         }
 
-    //     public async Task<Comment> updateComment(Client client,string id, Comment comment){
-    //         //put Comment
-    //         try{
-    //             HttpResponseMessage response = await client.PutAsJsonAsync("comments/"+ id, comment);
-    //             response.EnsureSuccessStatusCode();
+         public async Task<TModel> UpdateAsync(string id, TModel model){
+             //put Comment
+             try{
+                  var requestURL = $"{typeof(TModel).Name.ToLower()}s";
+                 HttpResponseMessage response = await client.PutAsJsonAsync($"{requestURL}/{id}", model);
+                 response.EnsureSuccessStatusCode();
```

Comment:
Change this condition to avoid throw an exception, instead, you can use response.IsSuccessStatusCode property. If this property is false, you can return a null or default instance.
This also can allow us to remove the Console.WriteLine("... saver/updated") message in Add an Update methods